moveit_core/planning_scene/include/moveit/planning_scene/planning_scene.h

```diff
@@ -983,6 +983,9 @@ class PlanningScene : private boost::noncopyable, public std::enable_shared_from
   bool processCollisionObjectRemove(const moveit_msgs::CollisionObject& object);
   bool processCollisionObjectMove(const moveit_msgs::CollisionObject& object);
 
+  /* convert Pose msg to Eigen::Isometry, normalizing the quaternion part if necessary. */
```

Comment:
doxygen requires `/** ... */` or `///`
```suggestion
  /** convert Pose msg to Eigen::Isometry, normalizing the quaternion part if necessary. */
```